contracts/zero-ex/contracts/src/transformers/bridges/mixins/MixinSushiswap.sol

```diff
@@ -0,0 +1,79 @@
+
+/*
+
+  Copyright 2020 ZeroEx Intl.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+pragma solidity ^0.6.5;
+pragma experimental ABIEncoderV2;
+
+import "@0x/contracts-erc20/contracts/src/v06/LibERC20TokenV06.sol";
+import "@0x/contracts-erc20/contracts/src/v06/IERC20TokenV06.sol";
+import "./MixinAdapterAddresses.sol";
+import "./MixinUniswapV2.sol";
+
+contract MixinSushiswap is
+    MixinAdapterAddresses
+{
+    using LibERC20TokenV06 for IERC20TokenV06;
+
+    /// @dev Mainnet address of the `SushiswapV2Router02` contract.
+    IUniswapV2Router02 private immutable SUSHISWAP_V2_ROUTER;
+
+    constructor(AdapterAddresses memory addresses)
+        public
+    {
+        SUSHISWAP_V2_ROUTER = IUniswapV2Router02(addresses.sushiswapV2Router);
+    }
+
+    function _tradeSushiswap(
```

Comment:
How about we drop this contract, reuse `MixinUniswapV2`, modify it to also store the sushi router, and we can pass in `bool isSushi` to this function? Should make deployments a bit cheaper.